rmp-serde/src/lib.rs

```diff
@@ -79,6 +79,19 @@ pub mod config;
 pub mod decode;
 pub mod encode;
 
+/// Name of Serde newtype struct to Represent Msgpack's Ext
+/// Msgpack Ext: Ext(tag, binary)
+/// Serde data model: _ExtStruct((tag, binary))
+/// Example Serde impl for custom type:
+///
+/// #[derive(Debug, PartialEq, Serialize, Deserialize)]
```

Comment:
To have this display correctly, I think we need to surround the code sample with a doc comment. Just adding three lines with the triple back-ticks (`) should do this.

If the code won't compile as a doc test, then making the first line

    ```ignore

will disable that.